lambda_function.py

```diff
@@ -0,0 +1,48 @@
+import os
+import json
+import urllib3
+import boto3
+from urllib3.exceptions import ConnectTimeoutError
+from boto3 import resource
+from botocore.exceptions import ClientError
+from boto3.dynamodb.conditions import Key, Attr
+
+http = urllib3.PoolManager()
+client = boto3.client('events', region_name="eu-west-2")
+
+
+def get_item_with_id(table, primary_key, primary_key_value):
+    response = table.query(KeyConditionExpression=Key(primary_key).eq(primary_key_value))
+    items = response["Items"]
+
+    if len(items) == 0:
+        raise Exception(f"{len(items)} items have the '{primary_key}' primary key of '{primary_key_value}'")
+    else:
+        return items[0]
+
+
+def add_true_to_ingest_cc_attribute(table, primary_key, sort_key, item_with_id):
+    primary_key_value = item_with_id[primary_key]
+    sort_key_value = item_with_id[sort_key]
+
+    table.update_item(
+        Key={primary_key: primary_key_value, sort_key: sort_key_value},
+        UpdateExpression="SET ingested_CC = :ingestedCCValue",
+        ExpressionAttributeValues={":ingestedCCValue": "true"},
+        ConditionExpression=f"attribute_exists({primary_key})"
+    )
+
+
+def get_message_from_json_event(event):
+    return json.loads(event['Records'][0]['Sns']['Message'])
+
+
+def lambda_handler(event, context):
+    message_json_as_dict = get_message_from_json_event(event)
+    primary_key_value = message_json_as_dict["tableItemIdentifier"]
+    primary_key = "id"
+    sort_key = "batchId"
+
+    table = resource("dynamodb").Table(os.environ["DYNAMO_TABLE_NAME"])
```

Comment:
[boto3 is no longer developing the resource interfaces](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/resources.html#:~:text=Note,the%20client%20interface.). As we're using the version of boto3 baked-in to the runtime by AWS would it be more future proof to use the client interface?